pom.xml

```diff
@@ -23,7 +23,7 @@
     <groupId>org.wso2.carbon.connector</groupId>
     <artifactId>org.wso2.carbon.connector.amazons3</artifactId>
     <packaging>jar</packaging>
-    <version>1.0.11-SNAPSHOT</version>
+    <version>1.0.11</version>
```

Comment:
As per practice, in the master branch we have SNAPSHOT version. When we do a release we remove it and do the release.

Reply:
Fixed